### DISCUSSION TITLE
Click once application won't run on windows 7, 8 after allowing dpiAware

### DISCUSSION THREAD

**Discussion by markismail7** [MicrosoftEdge/WebView2Feedback#3930] [Q&A]
So we're trying to make sure that our application scale perfectly on 4K monitors. We successfully did that, but now the app will not run on windows 7,8,8.1. It can only run on windows 10,11.

So by uncommenting the code below, the app will not run on older windows. Is there anyway around it, where we can keep the dpiAware true for windows 10,11 and false for older versions so users with older version of windows can still run the app?

Note: i'm using .NET 4.7.2 Thanks

    <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
    <application>
    

    
    -->

    
    <supportedOS Id="{35138b9a-5d96-4fbd-8e2d-a2440225f93a}" />

    
    <supportedOS Id="{4a2f28e3-53b9-4441-ba9c-d69d4a4a6e38}" />

    
    <supportedOS Id="{1f676c76-80e1-4239-95bb-83d0f6d0da78}" />

    
    <supportedOS Id="{8e0f7a12-bfb3-4fe8-b9a5-48fd50a15a9a}" />

    </application>
    </compatibility>

    <application xmlns="urn:schemas-microsoft-com:asm.v3">
    <windowsSettings>
    <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>
    <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
    </windowsSettings>
    </application>

**Comment by markismail7** [comment]
Microsoft says: On older versions of Windows, the newer <dpiAwareness> tag will be ignored. So basically if I want my app to run on older versions of windows like 7,8, i need to 

remove 

    <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true</dpiAware>

and keep 

    <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
So it would like:

    <application xmlns="urn:schemas-microsoft-com:asm.v3">
    <windowsSettings>   
	  <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2</dpiAwareness>
    </windowsSettings>
    </application>
By doing that, my app can run on 4k monitors using Windows 10, 11 and run normally on older versions of windows. Unfortunally, i can't verify that till tommorrow. So i won't be able to sleep tonight lol

**Comment by champnic** [comment]
Hey Mark - Sorry for the delayed response. Did your fix work? Did you get some sleep? :P
